### PR TITLE
Translate transitions part within migration file to Spanish

### PR DESCRIPTION
**Pull request by FLarra**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -726,38 +726,40 @@ to:
...
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>Vue.transition</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplose de <code>Vue.transition</code>.</p>
 </div>
 {% endraw %}
 
-### Transition `stagger` Attribute <sup>removed</sup>
+### Transición atributo `stagger` <sup>removido</sup>
```
Comment (Tintef) [inline src/v2/guide/migration.md]: This looks a bit odd to me 😞, but I can't think of a better way to translate it 😢

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -726,38 +726,40 @@ to:
...
 
-If you need to stagger list transitions, you can control timing by setting and accessing a `data-index` (or similar attribute) on an element. See [an example here](transitions.html#Staggering-List-Transitions).
+Si necesita escalonar las transiciones de la lista, puede controlar el tiempo configurando y accediendo a un `data-index` (o atributo similar) en un elemento. Ver [un ejemplo aquí](transitions.html#Staggering-List-Transitions).
```
Comment (Tintef) [inline src/v2/guide/migration.md]: I think `Ver` should be `Vea`
```suggestion
Si necesita escalonar las transiciones de la lista, puede controlar el tiempo configurando y accediendo a un `data-index` (o atributo similar) en un elemento. Vea [un ejemplo aquí](transitions.html#Staggering-List-Transitions).
```

File: src/v2/guide/migration.md [hunk 3]
```diff
@@ -726,38 +726,40 @@ to:
...
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of the <code>transition</code> attribute. During your update, you can transition (pun very much intended) to the new staggering strategy as well.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplose del atributo <code>transition</code>. Durante su actualización, usted puede hacer la transición (juego de palabras muy intencionado) a la nueva estrategia asombrosa también.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Typo:

```suggestion
    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar ejemplos del atributo <code>transition</code>. Durante su actualización, usted puede hacer la transición (juego de palabras muy intencionado) a la nueva estrategia asombrosa también.
```

File: src/v2/guide/migration.md (hunk 1 above)
Comment (FLarra) [reply src/v2/guide/migration.md]: Yes, it's a hard one. Open for discussion I guess...

Comment (sxcamacho) [reply src/v2/guide/migration.md]: @Tintef @FLarra  Checking [the portuguese translation](https://br.vuejs.org/v2/guide/migration.html#Atributo-de-Transicao-stagger-removido), they are using `Atributo de Transición`. What do you think?

Comment (FLarra) [reply src/v2/guide/migration.md]: I like it. Let me change it. Thank you!

**Comment by FLarra** [comment]
@sxcamacho addressed your last comment, please check the latest commit

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**